src/pages/v2/p/[projectId]/index.page.tsx

```diff
@@ -48,20 +48,21 @@ export default function V2ProjectPage({
 }: InferGetStaticPropsType<typeof getStaticProps>) {
   return (
     <>
-      {metadata ? (
-        <SEO
-          title={metadata.name}
-          url={`${process.env.NEXT_PUBLIC_BASE_URL}v2/p/${projectId}`}
-          description={metadata.description}
-          twitter={{
-            card: 'summary',
-            creator: metadata.twitter,
-            handle: metadata.twitter,
-            image: metadata.logoUri,
-            site: metadata.twitter,
-          }}
-        />
-      ) : null}
+      <SEO
+        // Set known values, leave others undefined to be overridden
+        title={metadata?.name ? metadata.name : undefined}
+        url={`${process.env.NEXT_PUBLIC_BASE_URL}v2/p/${projectId}`}
+        description={metadata?.description ? metadata.description : undefined}
+        twitter={{
+          card: 'summary',
+          creator: metadata?.twitter ? metadata.twitter : undefined,
+          handle: metadata?.twitter ? metadata.twitter : undefined,
```

Comment:
```suggestion
          creator: metadata?.twitter,
          handle: metadata?.twitter,
```

or at the very least we can do `metadata?.twitter ?? undefined`